index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
+
+type ListenerFn = (...args: Array<any>) => void;
+
+/**
+  * Minimal `EventEmitter` interface that is molded against the Node.js
```

Comment:
Can you please "fix" the indentation of these comments? Indent the second line and the followings with 1 space.